proto/streamlit/proto/AudioInput.proto

```diff
@@ -0,0 +1,33 @@
+/**!
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.snowflake.apps.streamlit";
+option java_outer_classname = "AudioInputProto";
+
+import "streamlit/proto/LabelVisibilityMessage.proto";
+
+
+message AudioInput {
+  string id = 1;
+  string label = 2;
+  string help = 3;
+  string form_id = 4;
+  bool disabled = 5;
+  LabelVisibilityMessage label_visibility = 6;
```

Comment:
I think this is another aspect we should codify: whether or not to use `optional` I think we use it pretty inconsistently 😅